src/wallet/wallet.cpp

```diff
@@ -1479,16 +1479,20 @@ bool CWallet::LoadWalletFlags(uint64_t flags)
     return true;
 }
 
-bool CWallet::AddWalletFlags(uint64_t flags)
+void CWallet::InitWalletFlags(uint64_t flags)
 {
     LOCK(cs_wallet);
+
     // We should never be writing unknown non-tolerable wallet flags
     assert(((flags & KNOWN_WALLET_FLAGS) >> 32) == (flags >> 32));
+    // This should only be used once, when creating a new wallet - so current flags are expected to be blank
```

Comment:
In commit "Bugfix: Wallet: Document expectations for AddWalletFlags (now InitWalletFlags) correctly" (0cb6d2aec63aec76a517b8da621a3c53ab432632)

Might be better to say "initializing a newly created wallet" instead of "creating a new wallet". Just because of the `CWallet::Create` function "create" sometimes means load instead of newly create.